src/azure-cli/azure/cli/command_modules/containerapp/_transformers.py

```diff
@@ -18,6 +19,41 @@ def transform_containerapp_output(app):
     return result
 
 
+def clean_up_sensitive_values(response_json):
+    for container in safe_get(response_json, "properties", "template", "containers", default=[]):
+        if "env" in container:
+            for env in container["env"]:
+                del env["value"]
+
+    if "scale" in response_json["properties"]["template"]:
+        for rule in safe_get(response_json, "properties", "template", "scale", "rules", default=[]):
+            for (key, val) in rule.items():
+                if key != "name":
+                    val["metadata"] = dict((k, "") for k, v in val["metadata"].items())
```

Comment:
Object manipulation can be hard to understand. It's better to provide some examples of the real data structure.